lox_services/persistence/database/utils.py

```diff
@@ -141,3 +143,53 @@ def make_temporary_table(
     table_ref = client.get_table(table_ref)
     table_ref.expires = datetime.now(timezone.utc) + timedelta(hours=1)
     client.update_table(table_ref, ["expires"])  # API request
+
+
+def make_validate_country_code() -> Callable[[pd.DataFrame, str], None]:
+    """Namespace for all the valid country codes."""
+    valid_codes = {country.alpha_2 for country in pycountry.countries}
+
+    def inner_validate_country_code(
```

Comment:
@VovaViliLox  Can you add in the docstring the fact that NaN values don't return an error. It looks like it does with your current kne